src/test/java/io/prometheus/wls/rest/WebClientImplTest.java

```diff
@@ -31,7 +34,10 @@
     private static final char QUOTE = '"';
 
     /** A URL with a host guaranteed not to exist. */
-    private static final String UNDEFINED_HOST_URL = "http://mxyptlk/";
+    private static final String UNDEFINED_HOST_URL = "http://seriously-this-should-not-exist/";
```

Comment:
Perhaps this RFC maybe helpful: https://tools.ietf.org/html/rfc2606

Reply:
Very helpful, thanks